### ISSUE TITLE
Compare with cron_core ?

### ISSUE THREAD

**Issue by pillarsdotnet** [voxpupuli/puppet-cron#82]
I notice that [cron_core](https://github.com/puppetlabs/puppetlabs-cron_core) is included with [puppet-agent](https://puppet.com/docs/puppet/latest/cheatsheet_core_types.html#cron) whereas this module must be explicitly loaded.

Please provide a comparison between the two modules, noting which is preferred for different use-cases.

**Comment by mfuhrmann** [comment]
Yeah. The differences are not clear or good to find (I didn't find anything).

**Comment by alexjfisher** [comment]
The [cron](https://puppet.com/docs/puppet/5.5/types/cron.html) type (now in cron_core, but formally a builtin type), manages the crontabs of users.  eg. all `cron` resources with `user => root` will end up in a single crontab (`/var/spool/cron/root` on EL7 at least).

The type inserts some comments with warnings to not mess with the file manually.
```
# HEADER: This file was autogenerated at 2021-03-31 16:26:03 +0100 by puppet.
# HEADER: While it can still be managed manually, it is definitely not recommended.
# HEADER: Note particularly that the comments starting with 'Puppet Name' should
# HEADER: not be deleted, as doing so could cause duplicate cron jobs.
# Puppet Name: foo
0 2 * * 5 foo.sh
# Puppet Name: bar
* 15 1 * * bar.sh
```

In contrast, this module creates cron entries by creating separate files under `/etc/cron.d`.  I think some people prefer this approach.

PRs to the docs, (maybe a rationale section??), always welcome. :)
